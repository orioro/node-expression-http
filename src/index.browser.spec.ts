import * as PUBLIC_API from './index.browser'
// import { testCases } from '@orioro/jest-util'

test('browser / public api', () => {
  expect(Object.keys(PUBLIC_API)).toMatchSnapshot()
})
