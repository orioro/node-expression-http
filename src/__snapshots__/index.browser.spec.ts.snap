// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`browser / public api 1`] = `
Array [
  "httpFetchExpression",
]
`;
